File: 03.FrontendRTE/src/app/shared/architecture/command/aggregate/notimplemented.dto.ts
import { JsonMember, JsonObject } from '@upe/typedjson';
import { DTO } from '@shared/architecture';

@JsonObject()
export class NotImplementedDTO extends DTO<NotImplementedDTO> {
  @JsonMember({
    isRequired: true,
    name: 'msg',
    type: String,
  }) private _msg: string;

  constructor() {
    super();
    this.typeName = 'NotImplementedDTO';
    this._msg = '';
  }

  get msg(): string {
    return this._msg;
  }

  set msg(value: string) {
    this._msg = value;
  }
}

File: 04.BackendRTE/src/main/java/de/montigem/be/dtos/rte/NotImplementedDTO.java
/*
 *  (c) Monticore license: https://github.com/MontiCore/monticore
 */

package de.montigem.be.dtos.rte;

public class NotImplementedDTO extends DTO {

  private String msg;

  protected NotImplementedDTO() {
    super("NotImplementedDTO");
  }

  public NotImplementedDTO(String msg) {
    this();
    this.msg = msg;
  }
}
